src/javascript/core/send.js

```diff
@@ -7,6 +7,7 @@ const settings = require('./settings');
 const utils = require('../utils');
 const Queue = require('./queue');
 const transports = require('./transports');
+const ie11 = function () { return !!window.MSInputMethodContext && !!document.documentMode; };
```

Comment:
```
const isIe11 = !!window.MSInputMethodContext && !!document.documentMode;
```

 - execute once on startup and clearer variable name

Reply:
I've updated the variable name. Having within a function makes the tests pass.